src/components/RadioButton/RadioButton.st.css

```
@@ -198,10 +198,26 @@
 
 /* Create the indicator (the dot/circle - hidden when not checked) */
 
+.root:box:hover:checked .wrapper {
+    border-color: applyOpacity(value(CurrentIconColor), 1);
+}
+
+.root:box:hover:not(:checked) .wrapper {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+}
+
+.root:box:focus-within .wrapper {
+    border-color: applyOpacity(value(CurrentIconColor), 1);
```

should this be:
```suggestion
    border-color: applyOpacity(value(CurrentBorderColor), 1);
```
?

---

deleted ^ as its included in 
.root:hover .radioIcon {
    border-color: applyOpacity(value(CurrentBorderColor), 1);
}
(and the box does not have opacity at all when it's checked)